### ISSUE TITLE
Bug: Proton shows up as a game in library

### ISSUE THREAD

**Issue by ToughGuyKunio** [ValveSoftware/steam-for-linux#6906]
Proton shows up as a game in "recent games" under library.

Steam ver: 2020-02-11
OS: Manjaro Linux

**Comment by emmandyar** [comment]
I'm having this issue as well. Shows proton instead of the game (in my case, Doom (2016), 
Built: Feb 11 2020, at 19:27:00
Steam API: v020
Steam Package versions: 1581460722

OS: Linux Mint 19.3 cinnamon
Kernel: 5.3.0-40

**Comment by kisak-valve** [comment]
Hello, is anyone still experiencing this issue on an up to date system?

**Comment by FabioLolix** [comment]
Yes, is still showing, Arch Linux updated, Steam client just updated

**Comment by X6205** [comment]
Well.. i can confirm that this is still unresolved issue.. and annoying one. Like how hard can be hide from recent games linux runtimes? How is possible that same issue is not present in Windows. I dont see in Windows Steam client in recent games steam runtime redistributables, but in linux are runtimes like Proton or Sniper recognized as a games..

**Comment by X6205** [comment]
I finally get it. You are aiming for 5 years anniversary on this bugreport before you start working on it..

![Snímka obrazovky zo dňa 2024-11-09 18-53-10](https://github.com/user-attachments/assets/c9a114bb-a596-4c65-9a4a-5e08e18168a1)

**Comment by bpopejoy** [comment]
Seeing this on both Bazzite (Fedora) and pop os.

**Comment by Armandik** [comment]
this is still an issue on my system.

Arch Linux with kernel 6.17.9-arch1-1

Steam Beta Branch:  Stable Client
Steam Version:  1763795278
Steam Client Build Date:  Sat, Nov 22 8:00 AM UTC -08:00
Steam Web Build Date:  Sat, Nov 22 7:35 AM UTC -08:00
Steam API Version:  SteamClient023
